packages/iTwinUI-react/src/core/Footer/Footer.tsx

```diff
@@ -67,74 +108,75 @@ const footerTranslations: TitleTranslations = {
  * @example <caption>Changing a url</caption>
  * <Footer customElements={(defaultElements) => defaultElements.map(element => ({ ...element, url: element.key === 'privacy' ? customPrivacyUrl : element.url }))} />
  */
-export const Footer = (props: FooterProps) => {
-  const { customElements, translatedTitles, className, ...rest } = props;
+export const Footer = Object.assign(
+  (props: FooterProps) => {
+    const {
+      children,
+      customElements,
+      translatedTitles,
+      className,
+      ...rest
+    } = props;
 
-  useTheme();
+    useTheme();
 
-  const titles = { ...footerTranslations, ...translatedTitles };
-  const defaultElements: FooterElement[] = [
-    {
-      key: 'copyright',
-      title: `© ${new Date().getFullYear()} Bentley Systems, Incorporated`,
-    },
-    {
-      key: 'termsOfService',
-      title: titles.termsOfService,
-      url:
-        'https://connect-agreementportal.bentley.com/AgreementApp/Home/Eula/view/readonly/BentleyConnect',
-    },
-    {
-      key: 'privacy',
-      title: titles.privacy,
-      url: 'https://www.bentley.com/en/privacy-policy',
-    },
-    {
-      key: 'termsOfUse',
-      title: titles.termsOfUse,
-      url:
-        'https://www.bentley.com/en/terms-of-use-and-select-online-agreement',
-    },
-    {
-      key: 'cookies',
-      title: titles.cookies,
-      url: 'https://www.bentley.com/en/cookie-policy',
-    },
-    {
-      key: 'legalNotices',
-      title: titles.legalNotices,
-      url: 'https://connect.bentley.com/Legal',
-    },
-  ];
+    const titles = { ...footerTranslations, ...translatedTitles };
+    const translatedElements = defaultFooterElements.map((element) => {
+      if (element.key && titles.hasOwnProperty(element.key)) {
+        const key = element.key as keyof TitleTranslations;
+        return {
+          ...element,
+          title:
+            element.key && titles.hasOwnProperty(element.key)
+              ? titles[key]
+              : element.title,
```

Comment:
we already checked for this condition so `element.title` will never be used

if typescript is complaining maybe we can use null assertion `!` rather than adding more code

Reply:
Ups, leftovers, not needed.